desktop/apps/partnerships/templates/mobile/hero_unit.jade

```diff
@@ -1,9 +1,7 @@
 .gallery-partnerships-hero-unit-header
-  .grid-2-up
-    .grid-item
-      h1!= hero.heading
-    .grid-item
-      .preamble!= hero.preamble
+  br
+  h1!= hero.heading
+  .preamble!= hero.preamble
```

Comment:
Realized it's from previous  implementation, but maybe we don't need to "unescape" heading and preamble here?